cmd/start.go

```diff
@@ -114,7 +114,10 @@ func Start(c *cli.Context) error {
 		return fmt.Errorf("could not check the stake: [%v]", err)
 	}
 	if !hasMinimumStake {
-		return fmt.Errorf("stake is below the required minimum")
+		return fmt.Errorf(`no minimum KEEP stake or operator is not authorized to use it;
+please make sure:
+- operator address in the configuration is correct and it has KEEP tokens delegated
+- operator contract has been authorized to operate on stake`)
```

Comment:
Maybe it would be better to remove line breaks as logger outputs to plaintext and having this message printed across multiple lines may cause parsing problems.

Reply:
Good point, updated to a one-liner:

> Starting keep-ecdsa client...
> 13:21:36.514 CRITI  keep-main: no minimum KEEP stake or operator is not authorized to use it; please make sure the operator address in the configuration is correct and it has KEEP tokens delegated and the operator contract has been authorized to operate on the stake main.go:55